platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/TemporaryFileBackedOutputStream.java

```diff
@@ -83,12 +101,30 @@ public void close() throws IOException {
         }
         try {
             fileBackedOutputStream.close();
-            fileBackedOutputStream.reset();
+            reset();
         } finally {
             isClosed = true;
         }
     }
 
+    /**
+     * Reset fileBackedOutputStream and retry if it fails.
+     */
+    @SuppressWarnings("unchecked")
+    private void reset() {
+
+        RetryPolicy retryPolicy = new RetryPolicy().retryOn(IOException.class)
+                .withBackoff(INITIAL_RETRY_SLEEP, MAX_DELAY, INITIAL_RETRY_SLEEP_UNIT)
+                .withMaxRetries(MAX_RETRY_ATTEMPTS);
+
+        Failsafe.with(retryPolicy)
+                .onFailedAttempt(throwable -> LOGGER.debug("failed to reset FBOS, will retry",
+                        throwable))
+                .onFailure(throwable -> LOGGER.debug("failed to reset FBOS", throwable))
```

Comment:
It just means that the temp file wouldn't get cleaned up. Not critical in the short term. But if it happens a lot, it could fill the disk. Do you still want it bumped to warning?